lib/index.js

```diff
@@ -122,7 +137,31 @@ function maybeSignWebBundle(webBundle, opts) {
   return signedWebBundle;
 }
 
-function validateIWAOptions(opts) {
+function maybeSetIwaDefaults(opts) {
+  if (
+    opts.integrityBlockSign.isIwa !== undefined &&
+    opts.integrityBlockSign.isIwa === false
```

Comment:
You technically don't really need the first part:
```suggestion
    opts.integrityBlockSign.isIwa === false
```